scipy/stats/tests/test_hypotests.py

```diff
@@ -1004,6 +1006,148 @@ def test_less_greater(self, input_sample, expected, alternative):
         )
 
 
+class TestBoschlooExact:
+    """Some tests to show that boschloo_exact() works correctly."""
+
+    ATOL = 1e-7
+    @pytest.mark.parametrize(
+        "input_sample,expected",
+        [
+            ([[2, 7], [8, 2]], (0.01852173, 0.009886142)),
```

Comment:
Weird, R is giving me 0.009886151 for the p-value. But I think it's small enough difference to ignore.